pkg/machine/qemu/machine.go

```diff
@@ -128,7 +129,12 @@ func (p *Provider) NewMachine(opts machine.InitOptions) (machine.VM, error) {
 	// Add network
 	// Right now the mac address is hardcoded so that the host networking gives it a specific IP address.  This is
 	// why we can only run one vm at a time right now
-	cmd = append(cmd, []string{"-netdev", "socket,id=vlan,fd=3", "-device", "virtio-net-pci,netdev=vlan,mac=5a:94:ef:e4:0c:ee"}...)
+	if runtime.GOOS != "windows" {
```

Comment:
Technically this could be not OS specific, but QEMU version specific switch - it should be possible to use podman machine w/o passing FDs on QEMU versions equal or newer than 7.2.0 on all platforms (no switch necessary at all). But I guess it is a no go yet as there are some older RHEL linuxes, which doesn't have latest QEMU, but still need to be supported, so, it is easier to keep it this way than replace it with a switch based on QEMU version.